lib/ex_json_schema/validator/multiple_of.ex

```diff
@@ -19,28 +19,29 @@ defmodule ExJsonSchema.Validator.MultipleOf do
     []
   end
 
-  defp do_validate(_, 0) do
-    []
-  end
+  @zero Decimal.new(0)
```

Comment:
Can you move this to the top before all function definitions? That's where the constants are in other modules.